dev/source/docs/testing-with-replay.rst

```diff
@@ -8,24 +8,22 @@ Introduction
 ============
 
 Replay is a program that takes a dataflash log file and replays it
-through the latest master code allowing for state estimation issues to
-be analysed with new code.
+through any branch's code allowing for state estimation issues to
+be analyzed using that branch's code instead of the code used while generating the dataflash log.
```

Comment:
A note here saying that the logging structures of the two branches must be the same belongs here....

Reply:
will add